33_prototypes/script.js

```diff
@@ -0,0 +1,36 @@
+// Task 1 __________________________________________
+
+const myArray = [1, 2, 3, 4, 5, 6, 7];
+
+class Degree {
+    constructor(arr) {
+        this.arr = arr;
+    }
+
+    pow(n) {

```

Comment:
the idea was to add these function to `Array.prototype.pow = ...`

but that's also okay